README.md

```diff
@@ -37,25 +36,25 @@ plugins:
 
 Most options are set under `custom.warmup` in the `serverless.yaml` file.
 
-* **folderName** (default `_warmup`)
-* **cleanFolder** (default `true`)
-* **name** (default `${service}-${stage}-warmup-plugin`)
-* **role** (default to role in the provider)
-* **tags** (default to serverless default tags)
-* **vpc** (default to vpc in provider, can be set to `false` to deploy the warmup function outside of VPC)
-* **memorySize** (default `128`)
-* **events** (default `- schedule: rate(5 minutes)`, can be any [Serverless event](https://serverless.com/framework/docs/providers/aws/events/))
-* **package** (default `package: { individually: true, exclude: ['**'], include: ['_warmup/**'] }`)
-* **timeout** (default `10` seconds)
-* **environment** (default to unset all package level environment variables. It can be used to set variables or to unset global variables by setting it to undefined. However, you should never have to change the default.)
-* **prewarm** (default `false`)
+* **folderName** Folder to temporarily store the generated code (defaults to `_warmup`)
+* **cleanFolder** Whether to automatically delete the generated code folder. you might want to keep it if you are doing some custom packaging (defaults to `true`)
+* **name** Name of the generated warmer lambda (defaults to `${service}-${stage}-warmup-plugin`)
+* **role** Role to apply to the warmer lambda (defaults to the role in the provider)
+* **tags** Tag to apply to the generated warmer lambda(defaults to the serverless default tags)
+* **vpc** The VPC and subnets in which to deploy. Can be any [Serverless VPC configuration](https://serverless.com/framework/docs/providers/aws/guide/functions#vpc-configuration) or be set to `false` in order to deploy the warmup function outside of a VPC (defaults to the vpc in the provider)
+* **memorySize** The memory to be assigned to the lambda (defaults to `128`)
+* **events** The event that triggers the warmer lambda. Can be any [Serverless event](https://serverless.com/framework/docs/providers/aws/events/) (defaults to `- schedule: rate(5 minutes)`)
+* **package** The package configuration. Can be any [Serverless package configuration](https://serverless.com/framework/docs/providers/aws/guide/packaging#package-configuration) (defaults to `{ individually: true, exclude: ['**'], include: ['_warmup/**'] }`)
+* **timeout** The timeout to apply to the lambda in seconds. (defaults to `10`)
+* **environment** The environment variables to pass to the warmer lambda. It can be used to set variables or to unset variables configured at the provider by setting them to `undefined`. However, you should never have to change the default. (defaults to unset all package level environment variables. )
```

Comment:
I think this could be simplified, maybe

> Can be used to set environment variables. You can also unset variables configured at the provider by setting them to `undefined`. However, you should never have to change the default.